py-kornia/tests/test_io.py

```diff
@@ -0,0 +1,94 @@
+from pathlib import Path
+import kornia_rs as K
+
+import torch
+import numpy as np
+
+# TODO: inject this from elsewhere
+DATA_DIR = Path(__file__).parents[2] / "tests" / "data"
```

Comment:
maybe just download it from https://github.com/kornia/data/ as a fixture under the the right scope

Reply:
this has to happen from rust/cargo side -- because it's used there too. Need to check about this. In any case this is  relatively small file.

Reply:
Oki